allennlp_models/modelcards/rc-transformer-qa.json

```diff
@@ -3,16 +3,16 @@
     "registered_model_name": "transformer_qa",
     "registered_predictor_name": null,
     "display_name": "Transformer QA",
-    "archive_file": "transformer-qa-2020-05-26.tar.gz",
+    "archive_file": "transformer-qa-2020-08-01.tar.gz",
```

Comment:
`08-01` is August first, two months ago. Is that really the date on that model?

Reply:
Oh darn, I uploaded it with the wrong date. Meant to put "10" not "08"